web/src/main/java/com/microsoft/applicationinsights/web/internal/cookies/HttpCookieFactory.java

```diff
@@ -46,12 +45,9 @@
      */
     public static Cookie generateSessionHttpCookie(
```

Comment:
So java SDK has a feature to generate session ID cookie if it wasn't set by JavaScript? How does it work around the problems mentioned here http://apmtips.com/blog/2016/07/09/tracking-users-in-api-apps/?

Reply:
This method does seem to be used anywhere but unit tests. As a matter of fact, this factory class is not used at all. We should remove it. Created #577